fe/fe-core/src/main/java/com/starrocks/common/Log4jConfig.java

```diff
@@ -61,11 +61,11 @@ public class Log4jConfig extends XmlConfiguration {
             "<Configuration status=\"info\" packages=\"com.starrocks.common\">\n" +
             "  <Appenders>\n" +
             "    <Console name=\"ConsoleErr\" target=\"SYSTEM_ERR\" follow=\"true\">\n" +
-            "      <PatternLayout pattern=\"%d{yyyy-MM-dd HH:mm:ss,SSS} %p (%t|%tid) [%C{1}.%M():%L] %m%n\"/>\n" +
```

Comment:
I think the timezone mostly `2024-03-06 19:50:46.049+08:00` like this?

Reply:
this is `Z` vs `X`, refer to https://docs.oracle.com/javase/8/docs/api/java/text/SimpleDateFormat.html 
![Screenshot 2024-03-06 at 8 05 17 PM](https://github.com/StarRocks/starrocks/assets/2017944/c91aaef0-bb46-4d2a-a334-4774ed5430b7)